.eslintrc.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
 		'plugin:md/prettier',
 	],
 	overrides: [
+		{
+			// Nothing to override for these files. This is here so eslint also checks for `.jsx` files by default
+			files: [ '**/*.jsx' ],
```

Comment:
In case you are wondering:
*  `.js` will be matched by default
* `.ts` and `.tsx` are declared in https://github.com/Automattic/wp-calypso/blob/7c7d77fc22d7859235cc42d959b66762ade485fd/.eslintrc.js#L98